apps/test-suite/tests/FileSystem.ts

```diff
@@ -37,38 +37,62 @@ export async function test({ describe, expect, it, ...t }) {
       });
     }
 
-    // This test fails on CI.
-    // it('Supports some operations on SAF picker files', async () => {
-    //   const saf = await FS.StorageAccessFramework.requestDirectoryPermissionsAsync();
-    //   if (!saf.granted) {
-    //     throw new Error();
-    //   }
-    //   const safDirectory = new Directory(saf.directoryUri);
-    //   safDirectory.list().forEach((sd) => {
-    //     sd.delete();
-    //   });
-    //   expect(safDirectory.list().length).toBe(0);
-
-    //   safDirectory.createFile('newFile', 'text/plain');
-    //   expect(safDirectory.list().length).toBe(1);
-
-    //   safDirectory.list().forEach((sd) => {
-    //     sd.delete();
-    //   });
-    //   expect(safDirectory.list().length).toBe(0);
-
-    //   const file = safDirectory.createFile('newFile', 'text/plain');
-    //   file.write('test');
-    //   expect(file.textSync()).toBe('test');
-    //   expect(file.bytesSync()).toEqual(new Uint8Array([116, 101, 115, 116]));
-    //   expect(file.base64Sync()).toBe('dGVzdA==');
-    //   const file2 = safDirectory.createFile('newFile2', 'text/plain');
-
-    //   file2.write(new Uint8Array([116, 101, 115, 116]));
-    //   expect(file2.textSync()).toBe('test');
-    //   expect(file2.size).toBe(4);
-    //   expect(safDirectory.size).toBe(8);
-    // });
+    describe('picker operations', () => {
+      // This test fails on CI.
+      const enabled = false;
+      if (enabled) {
```

Comment:
I think doing that like this is much better:
https://github.com/expo/expo/blob/da7cdc0d25890225d0c4639f7e0e4cc65e46ede2/apps/test-suite/tests/MediaLibrary.js#L102-L104